frontend/src/components/Quiz/Quizzes.jsx

```
@@ -0,0 +1,185 @@
+        });
+      })
+      .catch(err => {
+        console.error('Error submitting quiz:', err);
+        setError('Failed to submit quiz. Please try again.');
+        setLoading(false);
+      });
+  };
+
+  if (loading) return (
+    <div className="flex flex-col items-center justify-center my-12">
+      <div className="animate-spin rounded-full h-12 w-12 border-t-2 border-b-2 border-blue-500"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)